coderd/database/queries/workspacebuilds.sql

```diff
@@ -179,3 +179,62 @@ WHERE
 	wb.transition = 'start'::workspace_transition
 AND
 	pj.completed_at IS NOT NULL;
+
+-- name: GetWorkspaceBuildStatsByTemplates :many
+SELECT
+    w.template_id,
+	t.name AS template_name,
+	t.display_name AS template_display_name,
+	t.organization_id AS template_organization_id,
+    COUNT(*) AS total_builds,
+    COUNT(CASE WHEN pj.job_status = 'failed' THEN 1 END) AS failed_builds
+FROM
+    workspace_build_with_user AS wb
+JOIN
+    workspaces AS w ON
+    wb.workspace_id = w.id
+JOIN
+    provisioner_jobs AS pj ON
+    wb.job_id = pj.id
+JOIN
+    templates AS t ON
+	w.template_id = t.id
+WHERE
+    wb.created_at > @since
```

Comment:
```suggestion
    wb.created_at >= @since
```

Minor nit, "since" is inclusive, otherwise "after" works better.